tests/runner/src/utils/smoke/consume.rs

```diff
@@ -24,19 +24,21 @@ fn consume_wait_timeout() -> u64 {
 }
 
 /// verify consumers
-pub async fn validate_consume_message(client: Arc<Fluvio>, option: &TestOption, offsets: Offsets) {
-    if option.use_cli() {
+pub async fn validate_consume_message(client: Arc<Fluvio>, option: &TestCase, offsets: Offsets) {
+    let use_cli = option.vars.get("use_cli").unwrap_or(&"false".to_string()) == "true";
```

Comment:
since we are going extract arguments repeatedly, can we create type utility?  For example, here's what I would like to see:
`let use_cli = options.vars::<bool>("use_cli").expect("use_cli")`